packages/contracts-bedrock/justfile

```diff
@@ -56,6 +56,37 @@ test *ARGS: build-go-ffi
 test-dev *ARGS: build-go-ffi
   FOUNDRY_PROFILE=lite forge test {{ARGS}}
 
+# Runs upgrade path variant of contract tests.
+# Env Vars:
+# - ETH_RPC_URL must be set to a production (Sepolia or Mainnet) RPC URL.
+# - FORK_BLOCK_NUMBER can be set in the env, or else will fallback to the default block number.
+#   Reusing the default block number greatly speeds up the test execution time by caching the
+#   rpc call responses in ~/.foundry/cache/rpc. The default block will need to be updated
+#   when the L1 chain is upgraded.
```

Comment:
We should start a WIP runbook to document manual steps like this, so we don't forget them. Or this can be in a github issue, etc

Reply:
Good call: https://github.com/ethereum-optimism/optimism/issues/13410